internal/apps/dop/endpoints/pipeline.go

```diff
@@ -95,6 +95,11 @@ func (e *Endpoints) pipelineCreate(ctx context.Context, r *http.Request, vars ma
 	if err != nil {
 		return errorresp.ErrResp(err)
 	}
+
+	// also add project/org default config namespace
+	reqPipeline.ConfigManageNamespaces = append(reqPipeline.ConfigManageNamespaces, makeProjectDefaultLevelCmsNs(app.ProjectID)...)
```

Comment:
project namespace should be placed after org namespace, for priority reason.

Reply:
done